herbert_core/classes/MPIFramework/messages/FailedMessage.m

```diff
@@ -56,8 +68,35 @@
                 text =obj.payload_(1).error;
             end
         end
-        
     end
+    methods(Static,Access=protected)
+        function is_pers = get_persist_state()
+            % return the persistent state for a message
+            is_pers = true;
+        end
+%         function struc_r = replace_mexc_(struc)
```

Comment:
What is this disabled new code?

Reply:
The functionality is implemented by other means. The complex code (tested) is replaced by simple alternative. Removed now.